src/config/apollo-client.js

```diff
@@ -3,7 +3,7 @@ import ApolloClient from 'apollo-boost';
 export const client = new ApolloClient({
   uri: 'https://api.github.com/graphql',
   request: operation => {
-    const authHeader = process.env.REACT_APP_GITHUB_TOKEN;
+    const authHeader = process.env.NODE_ENV !== 'production' && process.env.REACT_APP_GITHUB_TOKEN;
```

Comment:
Umm if this condition `process.env.NODE_ENV !== 'production'` is false, it will return 'false' such as string and you're sending it in the headers. I think that's wrong or what you think @mcallegari10 ?

Reply:
Yes, maybe it’s better to set it without the `&&` and do the validation with an `if` 🤔